src/components/network/map-equipments.js

```diff
@@ -95,13 +96,14 @@ export default class MapEquipments {
             substationsIds
         );
         const isFullReload = substationsIds ? false : true;
-
         updatedSubstations
             .then((values) => {
-                this.updateSubstations(
-                    this.checkAndGetValues(values),
-                    isFullReload
-                );
+                if (currentNodeAtReloadCalling?.id === currentNode?.id) {
+                    this.updateSubstations(
```

Comment:
Maybe we could avoid this update. It looks like currentNodeAtReloadCalling is updated too soon so the test returns true and the treatment is not cancelled.